### PR TITLE
updating tests for github actions

### PR DESCRIPTION
**Pull request by scarrazza**

### CODE REVIEW

**Comment by Zaharid** [comment]
Please make the channel `https://packages.nnpdf.science/public`

**Comment by scarrazza** [comment]
It is already public, no?

**Comment by Zaharid** [comment]
I mean the URL should be that. The other one gives problems and should be removed from the server at some point, possibly right now.

File: nnpdfcpp/config/plotting.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#
+# Configuration file for plotting options
+#
+
+############################################################
+uset0         : false   # use t0 in chi2check, validphys
+pdfcteq       : CT14nlo
+pdfmstw       : MMHT2014nlo68cl
+plotformat    : eps   # root file format
+errorband     : true     # 1 for 1sigma, 0 for 68% c.l.
```
Comment (scarlehoff) [inline nnpdfcpp/config/plotting.yml:10]:
```suggestion
errorband     : true    # 1 for 1sigma, 0 for 68% c.l.
```

**Review by scarlehoff — APPROVED** [review]
Esperemos a que termine el que está corriendo ahora, pero esto debería funcionar.

**Comment by scarrazza** [comment]
Cool!

File: nnpdfcpp/config/plotting.yml (hunk 1 above)
Comment (Zaharid) [reply nnpdfcpp/config/plotting.yml]: @scarrazza @scarlehoff why do we have this file at all?

**Pull request merged**